README.md

```diff
@@ -92,31 +88,28 @@ Press the call button to connect to Twilio
 
 <img height="500px" src="images/quickstart/voice_make_call.png">
 
-### <a name="bullet7"></a>7. Generate `google-services.json` from the Google Developer Console
-
-The Programmable Voice Android SDK uses Google Cloud Messaging push notifications to let your application know when it is receiving an incoming call. If you want your users to receive incoming calls, you’ll need to enable GCM in your application.
+### <a name="bullet7"></a>7. Generate `google-services.json`
 
-Follow the steps in the [Google Developers Console](https://developers.google.com/mobile/add). You must enter the correct Android package name, in this case  `com.twilio.voice.quickstart` .
+The Programmable Voice Android SDK uses Firebase Cloud Messaging push notifications to let your application know when it is receiving an incoming call. If you want your users to receive incoming calls, you’ll need to enable FCM in your application.
 
-<img width="700px" src="images/quickstart/create_choose_app.png">
+Follow the steps under **Use the Firebase Assistant** in the [Firebase Developers Guide](https://firebase.google.com/docs/android/setup). Once you connect and sync to Firebase successfully, you will be able to download the `google-services.json` for your application. 
 
+Login to Firebase console and make a note of generated `Server API Key` and `Sender ID` in your notepad. You will need them in the next step.
```

Comment:
I would add a note to re run the application once the google services file has been updated.